api/src/public/login/script.js

```diff
@@ -153,17 +179,20 @@ document.addEventListener('DOMContentLoaded', function() {
 
```

Comment:
What do when someone accesses this link while they're logged in? 
Do I wait for `checkSession` and try the token link after? Skip `checkSession` and just overwrite the cookie? 
Keeping `checkSession` is nice for when clicking the link the second time....

Reply:
I think I'd make it part of the `requestTokenLogin` call that just returns a redirect if the current session is valid, so remove the checkSession from the tokenLogin branch I guess?

Reply:
Done!